diff -Naur nss_wrapper-1.1.3.orig/src/nss_wrapper.c nss_wrapper-1.1.3/src/nss_wrapper.c
--- nss_wrapper-1.1.3.orig/src/nss_wrapper.c	2016-03-17 19:26:09.000000000 +0000
+++ nss_wrapper-1.1.3/src/nss_wrapper.c	2016-04-15 05:52:47.088000000 +0000
@@ -96,7 +96,16 @@
 # define NSS_STATUS_UNAVAIL     NSS_UNAVAIL
 # define NSS_STATUS_TRYAGAIN    NSS_TRYAGAIN
 #else
-# error "No nsswitch support detected"
+enum nss_status
+{
+  NSS_STATUS_TRYAGAIN = -2,
+  NSS_STATUS_UNAVAIL,
+  NSS_STATUS_NOTFOUND,
+  NSS_STATUS_SUCCESS,
+  NSS_STATUS_RETURN
+};
+
+typedef enum nss_status NSS_STATUS;
 #endif
 
 #ifndef PTR_DIFF
diff -Naur nss_wrapper-1.1.3.orig/tests/nss_nwrap.c nss_wrapper-1.1.3/tests/nss_nwrap.c
--- nss_wrapper-1.1.3.orig/tests/nss_nwrap.c	2013-12-20 08:10:18.000000000 +0000
+++ nss_wrapper-1.1.3/tests/nss_nwrap.c	2016-04-15 05:53:05.580000000 +0000
@@ -21,7 +21,16 @@
 # define NSS_STATUS_UNAVAIL     NSS_UNAVAIL
 # define NSS_STATUS_TRYAGAIN    NSS_TRYAGAIN
 #else
-# error "No nsswitch support detected"
+enum nss_status
+{
+  NSS_STATUS_TRYAGAIN = -2,
+  NSS_STATUS_UNAVAIL,
+  NSS_STATUS_NOTFOUND,
+  NSS_STATUS_SUCCESS,
+  NSS_STATUS_RETURN
+};
+
+typedef enum nss_status NSS_STATUS;
 #endif
 
 NSS_STATUS _nss_nwrap_setpwent(void);
